scripts/kind-e2e/e2e.sh

```diff
@@ -69,5 +69,7 @@ ${DAPPER_SOURCE}/bin/subctl benchmark throughput --intra-cluster ${KUBECONFIGS_D
 
 ${DAPPER_SOURCE}/bin/subctl benchmark throughput --verbose ${KUBECONFIGS_DIR}/kind-config-cluster1 ${KUBECONFIGS_DIR}/kind-config-cluster2
 
+${SCRIPTS_DIR}/e2e.sh "$@"
```

Comment:
@dfarrell07 This is necessary so the Go E2E tests in _test/e2e_ are executed via `make e2e`.

Reply:
@tpantelis Thanks, that makes a lot of sense. I guess I'm mostly just surprised this didn't exist already.

Reply:
These are the first Go E2E tests in the repo.